src/server/connection.rs

```diff
@@ -1482,8 +1485,49 @@ impl Connection {
         self.video_ack_required = lr.video_ack_required;
     }
 
+    #[cfg(all(feature = "flutter", target_os = "windows"))]
```

Comment:
again, server/connection.rs does nothing with UI. Your logic is totally wrong here, and confused by remote and local side.

I know you want to tell Sciter (controlling side) does not support mutipe rdp. But here is controlled side, not controlling side.

So you have to add Sciter support. Otherwise, there will be problem if controlled side is the newest version with multiple rdp session support, and controlling side is Sciter which will hang there after input password.